cmd/bootnode/bootnode.go

```diff
@@ -0,0 +1,201 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package bootnode
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	relaylog "github.com/ipfs/go-log/v2"
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p/core/peer"
+	rcmgr "github.com/libp2p/go-libp2p/p2p/host/resource-manager"
+	"github.com/libp2p/go-libp2p/p2p/protocol/circuitv2/relay"
+
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/log"
+	"github.com/obolnetwork/charon/app/version"
+	"github.com/obolnetwork/charon/app/z"
+	"github.com/obolnetwork/charon/p2p"
+)
+
+// Config defines the config of the bootnode.
+type Config struct {
+	DataDir       string
+	HTTPAddr      string
+	P2PConfig     p2p.Config
+	LogConfig     log.Config
+	AutoP2PKey    bool
+	P2PRelay      bool
+	MaxResPerPeer int
+	MaxConns      int
+	RelayLogLevel string
+}
+
+// Run starts an Obol libp2p-tcp-relay and udp-discv5 bootnode.
```

Comment:
```suggestion
// Run starts a libp2p-tcp-relay and udp-discv5 bootnode.
```
Since anyone can start a bootnode, no?